app/jinja_filters.py

```diff
@@ -373,6 +373,17 @@ def max_value(a, b):
         raise Exception(msg)
 
 
+@blueprint.app_template_filter()
+def language_urls(languages, current_language):
+    switcher = []
+    for language in languages:
+        if language[0] == current_language:
+            continue
```

Comment:
Not a big deal, but could this be made simpler?
```
return [l for l in languages if l[0] != current_language]
```

Reply:
ah yeh, I refactored some logic out and was lazy